ports/azure-core-cpp/vcpkg.json

```
@@ -37,7 +38,8 @@
           "default-features": false,
           "features": [
             "ssl"
-          ]
+          ],
+          "version>=": "7.44"
```

This version constraint will fail if users are using the curated catalog, because there is no entry for `7.44` in the curl version database: https://github.com/microsoft/vcpkg/blob/master/versions/c-/curl.json

```suggestion
          ]
```

---

Is it feasible for us to add a version entry for 7.44 to the curl version database?

I am curious to understand the rationale why it went down to 7.48 only:
https://github.com/microsoft/vcpkg/pull/15652

FYI @antkmsft, we'd want to make a similar change in https://github.com/Azure/azure-sdk-for-cpp/blob/a8d7a48a07a1eed8cc60dc00a7f618b6282c5eb7/sdk/core/azure-core/vcpkg/vcpkg.json#L44-L46

---

As policy, we don't backfill older versions into the public catalog because there's no reasonable way for us to test them (and ensure they're fit for use). Since 7.48.0 is the oldest version available in the public catalog, the two reasonable approaches are:
1. Don't specify a constraint -- all users will have 7.48 or newer if they're using the curated catalog and if a user is trying to use a custom older version they'll just use overrides anyway to force that version
2. Specify a constraint on the oldest version _with a compatible version scheme_ in the curated catalog (7.74.0#5).

I strongly recommend option 1.

---

@ahsonkhan, I'd do the following: apply this PR suggestion, and make a similar PR removing the very same line from vcpkg.json in our repo. No other work is necessary. I am glad it is known to us now, so we don't have to make a `port-version: 1` commit to vcpkg repo removing the version.

---

I forgot to add, after we commit the version removal, we'll need to run `vcpkg x-add-version --overwrite-version azure-core-cpp` and do a subsequent commit.

---

I made the updates I talked above

---

Thanks for the fix @antkmsft - @ras0219 please let us know if it looks good now. Thanks!